typings/index.d.ts

```diff
@@ -4340,6 +4340,12 @@ export type VerificationLevel = keyof typeof VerificationLevels;
 
 export type VoiceBasedChannelTypes = 'GUILD_VOICE' | 'GUILD_STAGE_VOICE';
 
+export interface WebhookClientData {
```

Comment:
This should be a union of two interfaces that define what props they should have (WebhookClientData = WebhookClientDataURL | WebhookClientDataIdWithToken)